DOCS/man/options.rst

```diff
@@ -4101,6 +4101,11 @@ Input
     Whether this applies depends on the VO backend and how it handles
     keyboard input. Does not apply to terminal input.
 
+``--native-touch=<yes|no>``
+    (Windows only)
```

Comment:
considering the mention of wayland below this is not strictly correct is it?
maybe take a short paragraph to explain that on some platforms you can choose to get mouse or touch events but wayland always receives only touch events